libyul/AsmCodeGen.h

```diff
@@ -45,12 +45,10 @@ class CodeGenerator
 	static void assemble(
 		Block const& _parsedData,
 		AsmAnalysisInfo& _analysisInfo,
-		eth::Assembly& _assembly,
+		dev::eth::Assembly& _assembly,
```

Comment:
Can this be replaced by the abstract assembly interface? (Perhaps in a later PR)